pkgs/desktops/budgie/budgie-control-center/default.nix

```diff
@@ -101,14 +102,13 @@ stdenv.mkDerivation rec {
     glib
     glib-networking
     gnome-desktop
-    gnome-online-accounts
     gnome.adwaita-icon-theme
     gnome.cheese
     gnome.gnome-bluetooth_1_0
     gnome.gnome-remote-desktop
     gnome.gnome-settings-daemon
     gnome.gnome-user-share
-    gnome.mutter
+    magpie
```

Comment:
Both are not needed to build b-c-c I imagine, I think we add mutter here mostly because wrapGAppsHook can pick up `/nix/store/2b1pi24s7nfrmpsdwp80m98yh3mqkylp-mutter-45.4/share/gsettings-schemas/mutter-45.4` so gsettings stuff works because the packages is properly wrapped instead of `NIX_GSETTINGS_OVERRIDES_DIR` is set (magpie does not provide those gsettings-schemas right now)